practice/blackjack/main.py

```diff
@@ -0,0 +1,120 @@
+"""Blackjack playing game."""
+
+from hand import *
+from deck import *
+from player import *
+from card import *
```

Comment:
I consider `import *` harmful. Yes, in this case you know all of the names you're importing, but if it's someone else's code, you might clobber one of your other module's functions and increases the chance a typo will work in your code since it was a name you mistakenly imported.